.env.template

```diff
@@ -0,0 +1,2 @@
+export K8S_CLUSTER_ROOT=<your_cluster_root_path>
+export POD_NAMESPACE=$(shell kubectl config view --minify -o jsonpath='{..namespace}')
```

Comment:
Is POD_NAMESPACE used?

Reply:
Yes. It is used generally for the makefiles